### PR TITLE
Updated Srophe codebase with Usaybia specific code, images, fonts and …

### PR DESCRIPTION
**Pull request by wsalesky**
…pages.

### CODE REVIEW

**Comment by wsalesky** [comment]
@nathangibson This version is compatible and tested in eXist-db 6.0.1, which addresses the log4j issue as well as some memory leaks within eXist-db. There are some small differences in display/functionality but we can address those as we move forward with development. Please test the new version and let me know what you think.

**Review by nathangibson — COMMENTED** [review]
Thanks @wsalesky ! Going through this now. 

It looks like `<?xml version="1.0" encoding="UTF-8"?>` was removed from every XML file? Was this intentional?

**Comment by wsalesky** [comment]
Hum, no that was not intentional.  What files in particular?

**Comment by nathangibson** [comment]
If you take a look at "Files changed" you'll see 191 and that this is the most common change, perhaps it was executed for all HTML and XML files, e.g., error-page.html https://github.com/usaybia/srophe/pull/1/files#diff-613c2df7fc98ad55691eba0ad274c3562e0eab1ba3f78e442d2b6b3e274047cf

**Comment by nathangibson** [comment]
@wsalesky I thought (much too late) about the fact that we've mostly been working from the development branch in usaybia/srophe-exist-app but I didn't discuss this with you (yikes).

Going through and comparing a bit, it looks like you got at least some of the code for this refactor from the development branch?

**Comment by nathangibson** [comment]
You can just let me know the status and we can discuss to figure out how to proceed.

**Comment by wsalesky** [comment]
@nathangibson I worked of the development branch of usaybia/srophe-eXist-app  I think I caught all the new features, but let me know if you think I missed something. Also, I suppose we should correct the issue with the missing XML declaration. It does not cause a problem code wise, but seems like it will cause unnecessary problems with merges down the line. If you have time to do it that would be great, otherwise I may be able to get to it tomorrow, and can absolutely get to it by Monday.

**Comment by nathangibson** [comment]
> @nathangibson I worked of the development branch of usaybia/srophe-eXist-app

@wsalesky Whew! Thank you for being on top of that. 

> I think I caught all the new features, but let me know if you think I missed something. 

OK, I'm taking a look. I'll probably make one or more pull requests for merging into this branch so you can review stuff. A couple general questions: There seem to be some differences with d3xquery files, with modules/*.xql files and with persons/*.html files. Have you already checked those? Just wondering if I should include them in the PR; they might be just old code rather than custom content.

> Also, I suppose we should correct the issue with the missing XML declaration. It does not cause a problem code wise, but seems like it will cause unnecessary problems with merges down the line. If you have time to do it that would be great, otherwise I may be able to get to it tomorrow, and can absolutely get to it by Monday.

Yes, I'll let you know if I get to it :-)

File: CTS/cts-registry.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <cts-registry>
```
Comment (nathangibson) [inline CTS/cts-registry.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<cts-registry>
```

File: CTS/index.html [hunk 2]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html xmlns="http://www.w3.org/1999/xhtml">
```
Comment (nathangibson) [inline CTS/index.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
```

File: about.html [hunk 3]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline about.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: access-config.xml [hunk 4]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <config>
```
Comment (nathangibson) [inline access-config.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<config>
```

File: api-documentation/index.html [hunk 5]
```diff
@@ -1,6 +1,15 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline api-documentation/index.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: banner.html [hunk 6]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" class="image-banner">
```
Comment (nathangibson) [inline banner.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" class="image-banner">
```

File: bibl/browse.html [hunk 7]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```
Comment (nathangibson) [inline bibl/browse.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```

File: bibl/facet-def.xml [hunk 8]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <facets xmlns="http://expath.org/ns/facet">
```
Comment (nathangibson) [inline bibl/facet-def.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<facets xmlns="http://expath.org/ns/facet">
```

File: bibl/index.html [hunk 9]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```
Comment (nathangibson) [inline bibl/index.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```

File: bibl/search.html [hunk 10]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```
Comment (nathangibson) [inline bibl/search.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/bibl.html" data-template-at="content">
```

File: browse.html [hunk 11]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline browse.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: build.xml [hunk 12]
```diff
@@ -1,11 +1,10 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project default="xar" name="srophe">
+<project default="xar" name="usaybia">
```
Comment (nathangibson) [inline build.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<project default="xar" name="usaybia">
```

File: collection.xconf [hunk 13]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <collection xmlns="http://exist-db.org/collection-config/1.0">
```
Comment (nathangibson) [inline collection.xconf]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<collection xmlns="http://exist-db.org/collection-config/1.0">
```

File: d3xquery/index.html [hunk 14]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html>
```
Comment (nathangibson) [inline d3xquery/index.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html>
```

File: documentation/editors.xml [hunk 15]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <tei:TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0" xml:lang="en">
```
Comment (nathangibson) [inline documentation/editors.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<tei:TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:tei="http://www.tei-c.org/ns/1.0" xml:lang="en">
```

File: documentation/index.html [hunk 16]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline documentation/index.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: documentation/syriaca-tei-main.odd [hunk 17]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <tei:TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" xmlns:tei="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0" xml:lang="en">
```
Comment (nathangibson) [inline documentation/syriaca-tei-main.odd]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<tei:TEI xmlns="http://www.tei-c.org/ns/1.0" xmlns:sch="http://purl.oclc.org/dsdl/schematron" xmlns:tei="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0" xml:lang="en">
```

File: documentation/wiki.html [hunk 18]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline documentation/wiki.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: error-page.html [hunk 19]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" class="templates:surround?with=templates/page.html&amp;at=content">
```
Comment (nathangibson) [inline error-page.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" class="templates:surround?with=templates/page.html&amp;at=content">
```

File: expath-pkg.xml [hunk 20]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<package xmlns="http://expath.org/ns/pkg" name="http://syriaca.org/srophe" abbrev="srophe" version="2.01" spec="1.0">
-    <title>The Srophé web application</title>
+<package xmlns="http://expath.org/ns/pkg" name="http://srophe.org/usaybia" abbrev="usaybia" version="0.8.0-dev" spec="1.0">
```
Comment (nathangibson) [inline expath-pkg.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<package xmlns="http://expath.org/ns/pkg" name="http://srophe.org/usaybia" abbrev="usaybia" version="0.8.0-dev" spec="1.0">
```

File: facet-def.xml [hunk 21]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <facets xmlns="http://expath.org/ns/facet">
```
Comment (nathangibson) [inline facet-def.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<facets xmlns="http://expath.org/ns/facet">
```

File: index.html [hunk 22]
```diff
@@ -1,94 +1,102 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline index.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: persons/browse.html [hunk 23]
```diff
@@ -1,26 +1,37 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/persons.html" data-template-at="content">
+<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline persons/browse.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: persons/facet-def.xml [hunk 24]
```diff
@@ -1,3 +1,9 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <facets xmlns="http://expath.org/ns/facet">
```
Comment (nathangibson) [inline persons/facet-def.xml:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<facets xmlns="http://expath.org/ns/facet">
```

File: persons/index.html [hunk 25]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/persons.html" data-template-at="content">
+<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline persons/index.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: persons/record.html [hunk 26]
```diff
@@ -1,22 +1,22 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/geo.html" data-template-at="content">
+<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline persons/record.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: persons/search.html [hunk 27]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/persons.html" data-template-at="content">
+<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline persons/search.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" xmlns:xi="http://www.w3.org/2001/XInclude" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: repo.xml [hunk 28]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <meta xmlns="http://exist-db.org/xquery/repo">
```
Comment (nathangibson) [inline repo.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<meta xmlns="http://exist-db.org/xquery/repo">
```

File: resources/bootstrap/fonts/glyphicons-halflings-regular.svg [hunk 29]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg xmlns="http://www.w3.org/2000/svg">
```
Comment (nathangibson) [inline resources/bootstrap/fonts/glyphicons-halflings-regular.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg">
```

File: resources/fonts/glyphicons-halflings-regular.svg [hunk 30]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg xmlns="http://www.w3.org/2000/svg">
```
Comment (nathangibson) [inline resources/fonts/glyphicons-halflings-regular.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg">
```

File: resources/fonts/syrcomadiabene.svg [hunk 31]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomadiabene.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomantioch.svg [hunk 32]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomantioch.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcombatnan.svg [hunk 33]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcombatnan.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcombatnanbold.svg [hunk 34]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcombatnanbold.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomctesiphon.svg [hunk 35]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomctesiphon.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomedessa.svg [hunk 36]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomedessa.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomjerusalem.svg [hunk 37]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
```
Comment (nathangibson) [inline resources/fonts/syrcomjerusalem.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1">
```

File: resources/fonts/syrcomjerusalembold.svg [hunk 38]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomjerusalembold.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomjerusalemitalic.svg [hunk 39]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomjerusalemitalic.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: resources/fonts/syrcomkharput.svg [hunk 40]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <svg>
```
Comment (nathangibson) [inline resources/fonts/syrcomkharput.svg]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<svg>
```

File: modules/teiDocs/tei_all.xsd [hunk 41]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns:dcr="http://www.isocat.org/ns/dcr" xmlns:examples="http://www.tei-c.org/ns/Examples" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ns1="http://www.tei-c.org/ns/1.0" elementFormDefault="qualified" targetNamespace="http://www.tei-c.org/ns/1.0">
```
Comment (nathangibson) [inline modules/teiDocs/tei_all.xsd]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<xs:schema xmlns:dcr="http://www.isocat.org/ns/dcr" xmlns:examples="http://www.tei-c.org/ns/Examples" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ns1="http://www.tei-c.org/ns/1.0" elementFormDefault="qualified" targetNamespace="http://www.tei-c.org/ns/1.0">
```

File: places/browse.html [hunk 42]
```diff
@@ -0,0 +1,39 @@
+<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline places/browse.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: resources/xsl/bibliography.xsl [hunk 43]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nathangibson) [inline resources/xsl/bibliography.xsl]:
```suggestion

```

File: resources/xsl/citation.xsl [hunk 44]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nathangibson) [inline resources/xsl/citation.xsl]:
```suggestion

```

File: resources/xsl/tei2html.xsl [hunk 45]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nathangibson) [inline resources/xsl/tei2html.xsl]:
```suggestion

```

File: search-config-example.xml [hunk 46]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <search-config>
```
Comment (nathangibson) [inline search-config-example.xml]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<search-config>
```

File: search-facets.html [hunk 47]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline search-facets.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: search.html [hunk 48]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```
Comment (nathangibson) [inline search.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml" data-template="templates:surround" data-template-with="templates/page.html" data-template-at="content">
```

File: sparql/collection.xconf [hunk 49]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <collection xmlns="http://exist-db.org/collection-config/1.0">
```
Comment (nathangibson) [inline sparql/collection.xconf]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<collection xmlns="http://exist-db.org/collection-config/1.0">
```

File: sparql/index.html [hunk 50]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html>
```
Comment (nathangibson) [inline sparql/index.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html>
```

File: sparql/update-rdf.html [hunk 51]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html xmlns="http://www.w3.org/1999/xhtml">
```
Comment (nathangibson) [inline sparql/update-rdf.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml">
```

File: templates/bibl.html [hunk 52]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```
Comment (nathangibson) [inline templates/bibl.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```

File: templates/geo.html [hunk 53]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```
Comment (nathangibson) [inline templates/geo.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```

File: templates/page.html [hunk 54]
```diff
@@ -1,16 +1,18 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
+<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work">
```
Comment (nathangibson) [inline templates/page.html:2]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work">
```

File: templates/persons.html [hunk 55]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```
Comment (nathangibson) [inline templates/persons.html]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<html xmlns="http://www.w3.org/1999/xhtml" data-template="app:get-work" lang="en">
```

**Comment by nathangibson** [comment]
@wsalesky Did the `<?xml version="1.0" encoding="UTF-8"?>` tweaks and some simple image and PDF content. Calling it a day and will come back to other possible content changes another time.

**Comment by nathangibson** [comment]
@wsalesky Regarding the d3 resources, I've had a closer look now, and it looks like what you have here is more up to date (i.e. better abstracted e.g. with `$config:data-root`). I don't see any missing customizations there.

**Comment by wsalesky** [comment]
@nathangibson I did test the d3 against the running version, so it should be the most up to date version.

**Comment by nathangibson** [comment]
@wsalesky There was some custom code for app.xql at https://github.com/usaybia/srophe-eXist-app/blob/5e15491334806a287544e982c6a3380a42bd3b3d/modules/app.xql#L727 but as far as I can tell this was only for the image-text page alignment that we tried once and decided not to use.

**Comment by wsalesky** [comment]
@nathangibson Yes that is my memory as well.

**Comment by nathangibson** [comment]
@wsalesky With #6 I'm all done with my review. Will be merging in some Docker & Github action config as well.

**Comment by nathangibson** [comment]
Thanks for investing so much time into this the last week!

**Comment by wsalesky** [comment]
Great! Happy to get it up and running, and refactored with the Srophe code base, should make development work going forward much easier.

**Comment by nathangibson** [comment]
@wsalesky Am testing local docker images at the moment. Can you remind me, is it ok that some links are broken locally if I have the nav-base set as empty? E.g., clicking "Persons" on the home page takes me to `http://localhost:8080/persons/browse.html` (404) instead of `http://localhost:8080/exist/apps/usaybia/persons/browse.html` (which works). 

This should still work when deployed to the site, right?

**Comment by nathangibson** [comment]
@wsalesky PS if you want to try out a docker image, check out https://github.com/usaybia/srophe/pkgs/container/usaybia-srophe/22234982?tag=v0.8.0-dev-test

**Comment by wsalesky** [comment]
@nathangibson are you using NGIX to direct traffic to /exist/apps/usaybia ? if so then you should be fine. If not there is a setting you can change in eXist to set usaybia as the root app, although it has some issues in terms of using the dashboard.

**Comment by nathangibson** [comment]
@wsalesky Thanks, yes we are using NGINX to do that.

**Comment by wsalesky** [comment]
@nathangibson are you ready to merge this? Or are you still testing?

**Comment by nathangibson** [comment]
@wsalesky I’d still like to get #8 and #9 fixed so that the app is ready to deploy when we merge. But I think those are the only things outstanding.

**Pull request merged**